stream-transformation-test/stream-transformation-it/src/test/resources/liquibase/viewstore-db-changesets/001-event-log.table.changelog.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet id="event-store-001" author="TechPod"
```

Comment:
Do we need to duplicate these or can we just depend upon / maven copy the framework liquibase?